migrate.py

```diff
@@ -755,6 +770,13 @@ def assemble_collections(spec, args, target_github_org):
                     except LookupError as err:
                         docs_dependencies = []
                         logger.info('%s in %s', err, src)
+
+                    if plugin_type in ('callback',):
+                        try:
```

Comment:
This communicates the intent better:
```suggestion
                        with contextlib.suppress(LookupError):
```

Reply:
I have rewritten the PR to account for more cases than just callbacks so it's outdated now. Will pushed once tested.